changelog/pending/20250512--cli--update-package-add-help-to-mention-pulumi-yaml.yaml

```diff
@@ -0,0 +1,4 @@
+kind: improvement
+area: cli
+component: "cli/package"
+summary: "Update help text for `pulumi package add` to mention that it adds packages to Pulumi.yaml"
```

Comment:
This looks like it might have been generated by a different version of changie?

You can run `make changelog` from the repo root to run with the expected version.

Reply:
We are not using `changie` at all in this repo, so this changelog is not going to work.  Please always use `make changelog`.

Reply:
If it's easier and this change is minor enough, I can just kill the changelog entry altogether. Would that be preferable?

Reply:
Yeah it's either fixing it or removing it.  I'm fine with removing it, but one that doesn't work with our tooling might break our release process (or it might not, but I'd rather not find out during the release :) )

Reply:
File removed.